jdbc-adapter/virtualschema-jdbc-adapter/src/test/java/com/exasol/adapter/dialects/hive/HiveColumnMetadataReaderTest.java

```diff
@@ -0,0 +1,55 @@
+package com.exasol.adapter.dialects.hive;
+
+import static com.exasol.adapter.dialects.hive.HiveProperties.*;
+import static org.hamcrest.Matchers.*;
+import static org.junit.Assert.*;
+
+import java.sql.*;
+import java.util.*;
+
+import org.junit.jupiter.api.*;
+
+import com.exasol.adapter.*;
+import com.exasol.adapter.dialects.*;
+import com.exasol.adapter.jdbc.*;
+import com.exasol.adapter.metadata.*;
+
+class HiveColumnMetadataReaderTest {
+    private HiveColumnMetadataReader columnMetadataReader;
+
+    @BeforeEach
+    void beforeEach() {
+        this.columnMetadataReader = new HiveColumnMetadataReader(null, AdapterProperties.emptyProperties(),
```

Comment:
Move to `AbstractColumnMetadataReaderTest` (also from the Oracle counterpart) to reduce duplication.

Reply:
done